offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/DependencyManager.groovy

```diff
@@ -93,6 +97,7 @@ class DependencyManager {
     private SearchPersonViewModel searchPersonViewModel
     private CreatePersonViewModel createCustomerViewModelNewOffer
     private MaintainProductsViewModel maintainProductsViewModel
+    private MaintainProductsViewModel maintainProductsViewModelArchive
```

Comment:
We normally just distinct the naming according to the functionality: (createProductPresenter, archiveProductPresenter) 
```suggestion
    private MaintainProductsViewModel archiveProductsViewModel
```

Reply:
but it is the MaintainProductViewModel, I would like to keep it

Reply:
Yes but the other viewModels also are organised according to the use case and not their class 
See:
`private CreateProductViewModel copyProductViewModel
    private CreateProjectViewModel createProjectModel`